src/tcgwars/logic/impl/gen4/DiamondPearlPromos.groovy

```diff
@@ -644,95 +555,64 @@ public enum DiamondPearlPromos implements LogicCardInfo {
             }
           }
           move "Metallic Bolt", {
-            text "60 damage. Energy attached to Magnezone. If you do, this attack�s base damage is 120 instead of 60."
-            energyCost L, M, C, C, L, M
+            text "60 damage. You may discard a [L] Energy and a [M] Energy attached to Magnezone. If you do, this attack's base damage is 120 instead of 60."
+            energyCost L, M, C, C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 60
+
+              if (confirm(text)) {
+                damage 60
+                discardSelfEnergyAfterDamage(L, M)
+              }
```

Comment:
You must discard both a L and an M to do the extra damage. If you do not have a combination of energy that works here, this should not add the damage, nor ask for confirmation.